src/translate.js

```diff
@@ -1,15 +1,37 @@
 import React from 'react'
 import hoistNonReactStatics from 'hoist-non-react-statics'
+import PropTypes from 'prop-types'
 import I18nContext from './i18n-context'
 
 // higher order decorator for components that need `t`
-export default function translate() {
+export default function translate({ forwardRef } = {}) {
   return WrappedComponent => {
-    const _translate = props => (
-      <I18nContext.Consumer>
-        {t => <WrappedComponent {...props} t={t} />}
-      </I18nContext.Consumer>
-    )
+    const _translate = originalProps => {
+      const props = Object.assign({}, originalProps)
+      const forwardedRef = props.forwardedRef
+
+      delete props.forwardedRef
+
+      return (
+        <I18nContext.Consumer>
+          {t => <WrappedComponent {...props} t={t} ref={forwardedRef} />}
```

Comment:
Codacy has a fix for the issue: [Replace `t` with `(t)`](https://app.codacy.com/gh/nayaabkhan/react-polyglot/pullRequest?prid=7596186)
```suggestion
          {(t) => <WrappedComponent {...props} t={t} ref={forwardedRef} />}
```